### PR TITLE
jobs/bump-lockfile: run testiso in a stage

### PR DESCRIPTION
**Pull request by dustymabe**
Running it in a stage will cause it to show up properly in the Blue Ocean view.

### CODE REVIEW

**Comment by dustymabe** [comment]
Another option is to make the coreos-ci-lib function run it in a stage from within the function and then un-stage calls to it in this repo.

File: jobs/bump-lockfile.Jenkinsfile [hunk 1]
```diff
@@ -224,7 +224,9 @@ lock(resource: "bump-${params.STREAM}") {
                             // compressed one
                             shwrap("cosa compress --artifact=metal")
                         }
-                        kolaTestIso(cosaDir: env.WORKSPACE, arch: arch, marker: arch)
+                        stage("${arch}:Kola:TestISO") {
```
Comment (jlebon) [inline jobs/bump-lockfile.Jenkinsfile:227]: Should it be `${arch}:kola:testiso` for consistency with the `${arch}:kola` and `${arch}:kola:upgrade`?

**Review by jlebon — APPROVED** [review]
No strong opinion. I'd say it's probably cleaner to keep it here rather than coreos-ci-lib, because you can't "unstage" it as a caller if you don't want it for whatever reason (e.g. you're calling it as part of parallel branches and it'd mess up the BlueOcean view).

File: jobs/bump-lockfile.Jenkinsfile (hunk 1 above)
Comment (dustymabe) [reply jobs/bump-lockfile.Jenkinsfile]: yep. updated.

**Pull request merged**